Fix Master build

### Issue description

```
 % PREDICTIVE_TEST_SELECTION=false ./gradlew clean build docs --no-build-cache --refresh-dependencies

=========================================================
Standardized project names are disabled.
Consider enabling them with
    micronautBuild.useStandardizedProjectNames=true
in settings.gradle.

Then you will need to replace project dependencies from
   project(':foo')
to
   project(':micronaut-foo')
in your build scripts
=========================================================
Type-safe project accessors is an incubating feature.
Project accessors enabled, but root project name not explicitly set for 'buildSrc'. Checking out the project in different folders will impact the generated code and implicitly the buildscript classpath, breaking caching.

FAILURE: Build failed with an exception.

* Where:
Build file '/Users/sdelamo/github/micronaut-projects/micronaut-views/build.gradle' line: 2

* What went wrong:
An exception occurred applying plugin request [id: 'io.micronaut.build.internal.docs']
> Failed to apply plugin class 'io.micronaut.build.docs.JavadocAggregatorPlugin'.
   > A problem occurred configuring project ':test-suite'.
      > A problem occurred evaluating project ':test-suite'.
         > No such property: validation for class: org.gradle.accessors.dm.LibrariesForMn$MicronautLibraryAccessors

* Try:

```

--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -5,8 +5,9 @@ micronaut-test = "4.0.0-SNAPSHOT"
 
 micronaut-security = "4.0.0-SNAPSHOT"
 micronaut-serde = "2.0.0-SNAPSHOT"
+micronaut-validation = "4.0.0-SNAPSHOT"
 
-groovy = "4.0.6"
+groovy = "4.0.9"
 spock = "2.3-groovy-4.0"
 
 managed-freemarker = "2.3.32"
@@ -25,6 +26,7 @@ kotlinx-coroutines = "1.6.4"
 [libraries]
 micronaut-security = { module = "io.micronaut.security:micronaut-security-bom", version.ref = "micronaut-security" }
 micronaut-serde = { module = "io.micronaut.serde:micronaut-serde-bom", version.ref = "micronaut-serde" }
+micronaut-validation = { module = "io.micronaut.validation:micronaut-validation-bom", version.ref = "micronaut-validation" }
 
 managed-freemarker = { module = "org.freemarker:freemarker", version.ref = "managed-freemarker" }
 managed-handlebars = { module = "com.github.jknack:handlebars", version.ref = "managed-handlebars" }

--- a/gradle/wrapper/gradle-wrapper.properties
+++ b/gradle/wrapper/gradle-wrapper.properties
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0.2-bin.zip
 networkTimeout=10000
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists

--- a/settings.gradle
+++ b/settings.gradle
@@ -6,7 +6,7 @@ pluginManagement {
 }
 
 plugins {
-    id("io.micronaut.build.shared.settings") version "6.3.1"
+    id("io.micronaut.build.shared.settings") version "6.3.3"
 }
 
 enableFeaturePreview 'TYPESAFE_PROJECT_ACCESSORS'
@@ -29,8 +29,10 @@ include "test-suite-groovy"
 include "test-suite-kotlin"
 
 micronautBuild {
+    useStandardizedProjectNames=true
     addSnapshotRepository()
     importMicronautCatalog()
     importMicronautCatalog("micronaut-security")
     importMicronautCatalog("micronaut-serde")
+    importMicronautCatalog("micronaut-validation")
 }

--- a/test-suite-groovy/build.gradle
+++ b/test-suite-groovy/build.gradle
@@ -4,16 +4,17 @@ plugins {
 }
 
 dependencies {
+    testAnnotationProcessor(mnValidation.micronaut.validation.processor)
     testCompileOnly(mn.micronaut.inject.groovy)
 
-    testImplementation(mn.micronaut.validation)
+    testImplementation(mnValidation.micronaut.validation)
     testImplementation(mnTest.micronaut.test.spock)
 
     testImplementation(mnSerde.micronaut.serde.jackson)
     testImplementation(mn.micronaut.http.server.netty)
     testImplementation(mn.micronaut.http.client)
-    testImplementation projects.viewsSoy
-    testImplementation projects.viewsVelocity
+    testImplementation projects.micronautViewsSoy
+    testImplementation projects.micronautViewsVelocity
 }
 
 tasks.named('test') {

--- a/test-suite-kotlin/build.gradle
+++ b/test-suite-kotlin/build.gradle
@@ -5,7 +5,8 @@ plugins {
 }
 
 dependencies {
-    testImplementation(mn.micronaut.validation)
+    testAnnotationProcessor(mnValidation.micronaut.validation.processor)
+    testImplementation(mnValidation.micronaut.validation)
 
     testImplementation(libs.junit.jupiter.api)
     testImplementation(mnTest.micronaut.test.junit5)
@@ -17,9 +18,9 @@ dependencies {
     testImplementation(mnSerde.micronaut.serde.jackson)
     testImplementation(mn.micronaut.http.server.netty)
     testImplementation(mn.micronaut.http.client)
-    testImplementation projects.viewsSoy
+    testImplementation projects.micronautViewsSoy
     testImplementation(libs.kotlinx.coroutines.core)
-    testImplementation projects.viewsVelocity
+    testImplementation projects.micronautViewsVelocity
 }
 
 tasks.named('test') {

--- a/test-suite/build.gradle
+++ b/test-suite/build.gradle
@@ -5,9 +5,9 @@ plugins {
 
 dependencies {
     testAnnotationProcessor(mn.micronaut.inject.java)
-    testAnnotationProcessor(mn.micronaut.validation)
+    testAnnotationProcessor(mnValidation.micronaut.validation.processor)
 
-    testImplementation(mn.micronaut.validation)
+    testImplementation(mnValidation.micronaut.validation)
 
     testImplementation(libs.junit.jupiter.api)
     testImplementation(mnTest.micronaut.test.junit5)
@@ -24,10 +24,10 @@ dependencies {
     testImplementation(mnSerde.micronaut.serde.jackson)
     testImplementation(mn.reactor)
 
-    testImplementation projects.viewsVelocity
-    testImplementation projects.viewsCore
-    testImplementation projects.viewsSoy
-    testImplementation projects.viewsHandlebars
+    testImplementation projects.micronautViewsVelocity
+    testImplementation projects.micronautViewsCore
+    testImplementation projects.micronautViewsSoy
+    testImplementation projects.micronautViewsHandlebars
 }
 
 tasks.named('test') {

--- a/views-core/build.gradle
+++ b/views-core/build.gradle
@@ -3,9 +3,11 @@ plugins {
 }
 
 dependencies {
+    annotationProcessor(mnValidation.micronaut.validation.processor)
+
     compileOnly(mnSecurity.micronaut.security)
     compileOnly(mn.micronaut.management)
-    compileOnly(mn.micronaut.validation)
+    compileOnly(mmnValidation.micronaut.validationation)
 
     api(mn.micronaut.runtime)
     api(mn.micronaut.http)
@@ -13,11 +15,12 @@ dependencies {
 
     implementation(mn.reactor)
 
+    testAnnotationProcessor(mnValidation.micronaut.validation.processor)
     testImplementation(mnSerde.micronaut.serde.jackson)
     testImplementation(mn.micronaut.http.client)
     testImplementation(mn.micronaut.inject.java)
     testImplementation(mn.micronaut.http.server.netty)
     testImplementation(mn.micronaut.management)
-    testImplementation(mn.micronaut.validation)
+    testImplementation(mnValidation.micronaut.validation)
     testImplementation(mn.snakeyaml)
 }

--- a/views-core/build.gradle
+++ b/views-core/build.gradle
@@ -7,7 +7,7 @@ dependencies {
 
     compileOnly(mnSecurity.micronaut.security)
     compileOnly(mn.micronaut.management)
-    compileOnly(mmnValidation.micronaut.validationation)
+    compileOnly(mnValidation.micronaut.validation)
 
     api(mn.micronaut.runtime)
     api(mn.micronaut.http)

--- a/views-freemarker/build.gradle
+++ b/views-freemarker/build.gradle
@@ -3,25 +3,28 @@ plugins {
 }
 
 dependencies {
+    annotationProcessor(mnValidation.micronaut.validation.processor)
+
     implementation(libs.managed.freemarker)
 
     compileOnly(mn.micronaut.management)
-    compileOnly(mn.micronaut.validation)
+    compileOnly(mnValidation.micronaut.validation)
 
-    api projects.viewsCore
+    api projects.micronautViewsCore
     api(mn.micronaut.runtime)
     api(mn.micronaut.http)
     api(mn.micronaut.http.server)
 
     testCompileOnly(mn.micronaut.inject.groovy)
     testAnnotationProcessor(mn.micronaut.inject.java)
+    testAnnotationProcessor(mnValidation.micronaut.validation.processor)
 
     testImplementation(mnSerde.micronaut.serde.jackson)
     testImplementation(mn.reactor)
     testImplementation(mn.micronaut.http.client)
     testImplementation(mn.micronaut.http.server.netty)
     testImplementation(mn.micronaut.management)
-    testImplementation(mn.micronaut.validation)
+    testImplementation(mnValidation.micronaut.validation)
 
     testImplementation(mn.snakeyaml)
 }

--- a/views-handlebars/build.gradle
+++ b/views-handlebars/build.gradle
@@ -3,24 +3,28 @@ plugins {
 }
 
 dependencies {
-    api projects.viewsCore
+    annotationProcessor(mnValidation.micronaut.validation.processor)
+
+    api projects.micronautViewsCore
     api(libs.managed.handlebars)
 
     compileOnly(mn.micronaut.management)
-    compileOnly(mn.micronaut.validation)
+    compileOnly(mnValidation.micronaut.validation)
 
     api(mn.micronaut.runtime)
     api(mn.micronaut.http)
     api(mn.micronaut.http.server)
 
-    testCompileOnly(mn.micronaut.inject.groovy)
+    testAnnotationProcessor(mnValidation.micronaut.validation.processor)
     testAnnotationProcessor(mn.micronaut.inject.java)
+
+    testCompileOnly(mn.micronaut.inject.groovy)
 
     testImplementation(mnSerde.micronaut.serde.jackson)
     testImplementation(mn.micronaut.http.client)
     testImplementation(mn.micronaut.http.server.netty)
     testImplementation(mn.micronaut.management)
-    testImplementation(mn.micronaut.validation)
+    testImplementation(mnValidation.micronaut.validation)
 
     testImplementation(mn.snakeyaml)
 }

--- a/views-jte/build.gradle
+++ b/views-jte/build.gradle
@@ -6,7 +6,7 @@ plugins {
 dependencies {
     api(libs.managed.jte)
 
-    api projects.viewsCore
+    api projects.micronautViewsCore
     implementation(libs.managed.jte.kotlin)
     testImplementation(mnSerde.micronaut.serde.jackson)
     testImplementation(mn.reactor)

--- a/views-pebble/build.gradle
+++ b/views-pebble/build.gradle
@@ -2,24 +2,27 @@ plugins {
     id "io.micronaut.build.internal.views-module"
 }
 dependencies {
-    api projects.viewsCore
+    annotationProcessor(mnValidation.micronaut.validation.processor)
+
+    api projects.micronautViewsCore
     api(libs.pebble)
 
     compileOnly(mn.micronaut.management)
-    compileOnly(mn.micronaut.validation)
+    compileOnly(mnValidation.micronaut.validation)
 
     api(mn.micronaut.runtime)
     api(mn.micronaut.http)
     api(mn.micronaut.http.server)
 
     testCompileOnly(mn.micronaut.inject.groovy)
+    testAnnotationProcessor(mnValidation.micronaut.validation.processor)
     testAnnotationProcessor(mn.micronaut.inject.java)
 
     testImplementation(mnSerde.micronaut.serde.jackson)
     testImplementation(mn.micronaut.http.client)
     testImplementation(mn.micronaut.http.server.netty)
     testImplementation(mn.micronaut.management)
-    testImplementation(mn.micronaut.validation)
+    testImplementation(mnValidation.micronaut.validation)
 
     testImplementation(mn.snakeyaml)
 }

--- a/views-rocker/build.gradle
+++ b/views-rocker/build.gradle
@@ -14,16 +14,19 @@ rocker {
 }
 
 dependencies {
-    api projects.viewsCore
+    annotationProcessor(mnValidation.micronaut.validation.processor)
+
+    api projects.micronautViewsCore
     api(libs.managed.rocker.runtime)
     compileOnly(mn.micronaut.management)
-    compileOnly(mn.micronaut.validation)
+    compileOnly(mnValidation.micronaut.validation)
 
     api(mn.micronaut.runtime)
     api(mn.micronaut.http)
     api(mn.micronaut.http.server)
 
     testCompileOnly(mn.micronaut.inject.groovy)
+    testAnnotationProcessor(mnValidation.micronaut.validation.processor)
     testAnnotationProcessor(mn.micronaut.inject.java)
 
     testImplementation(mnSerde.micronaut.serde.jackson)
@@ -31,6 +34,6 @@ dependencies {
     testImplementation(mn.micronaut.http.client)
     testImplementation(mn.micronaut.http.server.netty)
     testImplementation(mn.micronaut.management)
-    testImplementation(mn.micronaut.validation)
+    testImplementation(mnValidation.micronaut.validation)
     testImplementation(mn.snakeyaml)
 }

--- a/views-soy/build.gradle
+++ b/views-soy/build.gradle
@@ -3,22 +3,25 @@ plugins {
 }
 
 dependencies {
-    api projects.viewsCore
+    annotationProcessor(mnValidation.micronaut.validation.processor)
+
+    api projects.micronautViewsCore
     api(libs.managed.soy)
     compileOnly(mn.micronaut.management)
-    compileOnly(mn.micronaut.validation)
+    compileOnly(mnValidation.micronaut.validation)
 
     api(mn.micronaut.runtime)
     api(mn.micronaut.http)
     api(mn.micronaut.http.server)
 
     testCompileOnly(mn.micronaut.inject.groovy)
+    testAnnotationProcessor(mnValidation.micronaut.validation.processor)
     testAnnotationProcessor(mn.micronaut.inject.java)
 
     testImplementation(mnSerde.micronaut.serde.jackson)
     testImplementation(mn.micronaut.http.client)
     testImplementation(mn.micronaut.http.server.netty)
     testImplementation(mn.micronaut.management)
-    testImplementation(mn.micronaut.validation)
+    testImplementation(mnValidation.micronaut.validation)
     testImplementation(mn.snakeyaml)
 }

--- a/views-thymeleaf/build.gradle
+++ b/views-thymeleaf/build.gradle
@@ -3,11 +3,13 @@ plugins {
 }
 
 dependencies {
-    api projects.viewsCore
+    annotationProcessor(mnValidation.micronaut.validation.processor)
+
+    api projects.micronautViewsCore
     api(libs.managed.thymeleaf)
     api(libs.thymeleaf.extras.java8time)
     compileOnly(mn.micronaut.management)
-    compileOnly(mn.micronaut.validation)
+    compileOnly(mnValidation.micronaut.validation)
 
     api(mn.micronaut.runtime)
     api(mn.micronaut.http)
@@ -15,6 +17,7 @@ dependencies {
     api(mn.micronaut.inject)
 
     testCompileOnly(mn.micronaut.inject.groovy)
+    testAnnotationProcessor(mnValidation.micronaut.validation.processor)
     testAnnotationProcessor(mn.micronaut.inject.java)
 
     testImplementation(mnSerde.micronaut.serde.jackson)
@@ -22,6 +25,6 @@ dependencies {
     testImplementation(mn.micronaut.http.client)
     testImplementation(mn.micronaut.http.server.netty)
     testImplementation(mn.micronaut.management)
-    testImplementation(mn.micronaut.validation)
+    testImplementation(mnValidation.micronaut.validation)
     testImplementation(mn.snakeyaml)
 }

--- a/views-velocity/build.gradle
+++ b/views-velocity/build.gradle
@@ -3,18 +3,20 @@ plugins {
 }
 
 dependencies {
+    annotationProcessor(mnValidation.micronaut.validation.processor)
     annotationProcessor(mn.micronaut.graal)
 
-    api projects.viewsCore
+    api projects.micronautViewsCore
     api(libs.managed.velocity.engine.core)
     compileOnly(mn.micronaut.management)
-    compileOnly(mn.micronaut.validation)
+    compileOnly(mnValidation.micronaut.validation)
 
     api(mn.micronaut.runtime)
     api(mn.micronaut.http)
     api(mn.micronaut.http.server)
 
     testCompileOnly(mn.micronaut.inject.groovy)
+    testAnnotationProcessor(mnValidation.micronaut.validation.processor)
     testAnnotationProcessor(mn.micronaut.inject.java)
 
     testImplementation(mnSerde.micronaut.serde.jackson)
@@ -23,6 +25,6 @@ dependencies {
     testImplementation(mn.micronaut.http.server.netty)
 
     testImplementation(mn.micronaut.management)
-    testImplementation(mn.micronaut.validation)
+    testImplementation(mnValidation.micronaut.validation)
     testImplementation(mn.snakeyaml)
 }

